### PR TITLE
remove ssh-agent from `dcos node ssh`

### PR DESCRIPTION
**Pull request by mgummelt**

### CODE REVIEW

**Comment by mgummelt** [comment]
@jsancio: This is blocking #270, so let's get this done first

**Comment by jsancio** [comment]
Okay. Looking at the PR.

File: dcos/http.py [hunk 1]
```diff
@@ -66,22 +66,21 @@ def request(method,
...
-                headers={'Accept': 'application/json'},
-                **kwargs)
-
-        logger.info(
-            'Sending HTTP [%r] to [%r]: %r',
-            request.method,
-            request.url,
-            request.headers)
+    if 'headers' not in kwargs:
+        kwargs['headers'] = {'Accept': 'application/json'}
```
Comment (jsancio) [inline dcos/http.py]: :+1:

File: cli/tests/integrations/test_task.py [hunk 2]
```diff
@@ -12,6 +11,7 @@
 from dcos.util import create_schema
 from dcoscli.task.main import _mesos_files, main
 
+import fcntl
```
Comment (jsancio) [inline cli/tests/integrations/test_task.py]: This fails CI.

**Comment by jsancio** [comment]
Looks good to me. :shipit: when CI passes.

**Pull request merged**